src/Workspaces/Core/Portable/Workspace/Solution/ProjectState.cs

```diff
@@ -484,7 +488,7 @@ public ProjectState UpdateName(string name)
             return this.With(projectInfo: this.ProjectInfo.WithName(name).WithVersion(this.Version.GetNewerVersion()));
         }
 
-        public ProjectState UpdateFilePath(string filePath)
+        public ProjectState UpdateFilePath(string? filePath)
```

Comment:
❔ Do we allow going from non-null value to null value here?

Reply:
Yes.